### PR TITLE
Test staging

### LINKED ISSUE #69: Staging refactor/bug fix
**Description:**

- Refactor the  staging branch to fix the bugs that were created after merging services.

### PR DESCRIPTION
**Pull request by abhisachdev17**
## Creating a new wallet
create

## Getting the wallet amount
amount

### CODE REVIEW

**Review by akshaysharma21 — APPROVED** [review]

File: src/backend/.gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 **/env/
```
Comment (ZackHolmberg) [inline src/backend/.gitignore:1]: **/env/
**/__pycache__/
**/.cache
src/**.pyc
tests/**.pyc
.pytest_cache
__init__.pyc
**/**.pyc

Add this and then we don't have to have any other .gitignores

File: src/backend/blockchain/src/routes.py [hunk 2]
```diff
@@ -25,20 +25,21 @@ def get_chain():
...
-    return "proof was hit. You sent:"+request.json
+    return "proof was hit. You sent:"+request.get_json()
 
 
 @app.route('/wallet/addWallet', methods=['POST'])
 def addWallet():
     try:
-        data = request.json
+        data = request.get_json()
+        print(data)
```
Comment (ZackHolmberg) [inline src/backend/blockchain/src/routes.py]: Remove this, I assume it was from debugging

File: src/backend/blockchain/src/routes.py [hunk 3]
```diff
@@ -58,10 +59,10 @@ def verifyAmount():
         return jsonify(err=str(e)), HttpCode.BAD_REQUEST
 
 
-@app.route('/wallet/balance', methods=['GET'])
+@app.route('/wallet/balance', methods=['POST'])
```
Comment (ZackHolmberg) [inline src/backend/blockchain/src/routes.py]: Change back to GET

File: src/backend/blockchain/tests/functional/test_routes.py [hunk 4]
```diff
@@ -78,12 +78,12 @@ def test_get_wallet_amount(test_client):
     data1 = {'walletId': "fake_wallet_id"}
     data2 = {'walletId': "non_existent_wallet_id"}
 
-    response = test_client.get(url, data=json.dumps(data1), headers=headers)
+    response = test_client.post(url, data=json.dumps(data1), headers=headers)
```
Comment (ZackHolmberg) [inline src/backend/blockchain/tests/functional/test_routes.py]: Change back to get

File: src/backend/blockchain/tests/functional/test_routes.py [hunk 5]
```diff
@@ -78,12 +78,12 @@ def test_get_wallet_amount(test_client):
...
 
     assert response.status_code == 200
     assert response.json['amount'] == 0
 
-    response = test_client.get(url, data=json.dumps(data2), headers=headers)
+    response = test_client.post(url, data=json.dumps(data2), headers=headers)
```
Comment (ZackHolmberg) [inline src/backend/blockchain/tests/functional/test_routes.py]: Change back to get

File: src/backend/services/.gitignore [hunk 6]
```diff
@@ -1,2 +1,3 @@
 **/env/
-**/__pycache__/
```
Comment (ZackHolmberg) [inline src/backend/services/.gitignore:2]: can remove this file with root one updated

**Review by ZackHolmberg — CHANGES_REQUESTED** [review]
Looks good, few small changes

**Review by ZackHolmberg — APPROVED** [review]

**Pull request merged**